tests/accuracy/main.rs

```diff
@@ -103,7 +103,7 @@ fn verify(filelines: &[Pos]) {
         loop {
             let sym = match symbols.next() {
                 Some(sym) => sym,
-                None => panic!("failed to find {}:{}", file, line),
+                None => panic!("{}", "failed to find {file}:{line}"),
```

Comment:
This will panic with the literal message `failed to find {file}:{line}` now without substituting `{file}` and `{line}`.

Reply:
@bjorn3 good catch, thanks!